lib/templates/fragments/header.php

```diff
@@ -47,91 +53,120 @@ function beans_favicon() {
-		'rel'  => 'Shortcut Icon',
-		'href' => $url, // Automatically escaped.
-		'type' => 'image/x-icon',
-	) );
-
+	beans_selfclose_markup_e(
+		'beans_favicon',
+		'link',
+		array(
+			'rel'  => 'Shortcut Icon',
+			'href' => $url, // Automatically escaped.
+			'type' => 'image/x-icon',
+		)
+	);
 }
 
 beans_add_smart_action( 'wp_head', 'beans_header_image' );
 /**
  * Print the header image css inline in the header.
  *
  * @since 1.0.0
+ *
+ * @return void
  */
 function beans_header_image() {
 
-	if ( ! current_theme_supports( 'custom-header' ) || ! ( $header_image = get_header_image() ) || empty( $header_image ) ) {
+	$header_image = get_header_image();
+
+	if ( ! current_theme_supports( 'custom-header' ) || ! $header_image || empty( $header_image ) ) {
 		return;
 	}
 
-	?><style type="text/css">
+	?>
+	<style type="text/css">
 		.tm-header {
 			background-image: url(<?php echo esc_url( $header_image ); ?>);
 			background-position: 50% 50%;
 			background-size: cover;
 			background-repeat: no-repeat;
 		}
-	</style><?php
-
+	</style>
+	<?php
 }
 
 beans_add_smart_action( 'beans_header', 'beans_site_branding' );
 /**
  * Echo header site branding.
  *
  * @since 1.0.0
+ *
+ * @return void
  */
 function beans_site_branding() {
-
-	beans_open_markup_e( 'beans_site_branding', 'div', array(
-		'class' => 'tm-site-branding uk-float-left' . ( ! get_bloginfo( 'description' ) ? ' uk-margin-small-top' : null ),
-	) );
-
-		beans_open_markup_e( 'beans_site_title_link', 'a', array(
-			'href'     => home_url(), // Automatically escaped.
-			'rel'      => 'home',
-			'itemprop' => 'headline',
-		) );
-
-			if ( $logo = get_theme_mod( 'beans_logo_image', false ) ) {
-				beans_selfclose_markup_e( 'beans_logo_image', 'img', array(
-					'class' => 'tm-logo',
-					'src'   => $logo, // Automatically escaped.
-					'alt'   => get_bloginfo( 'name' ), // Automatically escaped.
-				) );
-			} else {
+	beans_open_markup_e(
+		'beans_site_branding',
+		'div',
+		array(
+			'class' => 'tm-site-branding uk-float-left' . ( ! get_bloginfo( 'description' ) ? ' uk-margin-small-top' : null ),
+		)
+	);
+
+		beans_open_markup_e(
+			'beans_site_title_link',
+			'a',
+			array(
+				'href'     => home_url(), // Automatically escaped.
+				'rel'      => 'home',
+				'itemprop' => 'headline',
+			)
+		);
+
+			$logo = get_theme_mod( 'beans_logo_image', false );
+
+			if ( $logo ) { // @codingStandardsIgnoreLine - Generic.WhiteSpace.ScopeIndent.IncorrectExact.
+				beans_selfclose_markup_e(
+					'beans_logo_image',
+					'img',
+					array(
+						'class' => 'tm-logo',
+						'src'   => $logo, // Automatically escaped.
+						'alt'   => get_bloginfo( 'name' ), // Automatically escaped.
+					)
+				);
+			} else { // @codingStandardsIgnoreLine - Generic.WhiteSpace.ScopeIndent.IncorrectExact.
 				beans_output_e( 'beans_site_title_text', get_bloginfo( 'name' ) );
-			}
+			} // @codingStandardsIgnoreLine - Generic.WhiteSpace.ScopeIndent.IncorrectExact.
```

Comment:
Get into the habit of adding _why_ we are ignoring this rule.  Add the "why" after the rule:

```
// @codingStandardsIgnoreLine - Generic.WhiteSpace.ScopeIndent.IncorrectExact - Indentation is intentional to show HTML markup structure.
```